### DISCUSSION TITLE
autodl settings button missing from rutorrent

### DISCUSSION THREAD

**Discussion by Paladin-z** [swizzin/swizzin#705] [Q&A + Help] [answered]
After updating my server (running Ubuntu 18.04.1 LTS (GNU/Linux 4.9.0-15-amd64 x86_64) I started getting "Bad response from server: (502 [error,getplugins]) Bad Gateway" errors in rutorrent. 

Removing and reinstalling rutorrent resolved that problem and everything appears to be working except that the autodl settings button is missing from the rutorent interface.

rutorrent is running, autodl is running, irssi is running, attaching to their screens shows no issues.

Any help getting the settings menu back would be appreciated :)

**Comment by flying-sausages** [comment]
You might need to re-install the autodl plugin using `rtx` https://swizzin.ltd/scripts/rtx

**Reply by Paladin-z** [reply]
Thanks flying-sausages, I attempted to remove and reinstall autodl and ran into broken package errors in Ubuntu that wouldn't let me reonstall it, so now I'm trying to resolve those first.

**Comment by rblair2** [comment, marked as answer]
Hi, I'm having a similar issue. Have reinstalled ubuntu to debian and reinstalled swizzin and the issue still persists. 

Sudo box rtx doesn't have an option to install auto-dl plugin either

Edit: this is the error in rutorrent log: autodl-irssi: __init2: ex: Exception info: No content elem with id autodl-prefs-contents-upload

Edit2: apparently it's an issue with the new version of rutorrent, and reverting to an older version as per the last post here works https://github.com/autodl-community/autodl-irssi/issues/213
